### ISSUE TITLE
Network Training

### ISSUE THREAD

**Issue by genbing99** [crane-papercode/ICE-GAN#2]
Hi, can you upload the code for the network training and testing? Thanks.

**Comment by Hazel1021** [comment]
> Hi, can you upload the code for the network training and testing? Thanks.

Hello,have you get or implement the code for this network's training and testing?

**Comment by tvaranka** [comment]
I am thinking this code may also have the bug from NMER code (https://github.com/xiaobaishu0097/MEGC2019). If you look at the logs for training, you can see that the F1 goes from 0.2 to 0.8 in a single epoch.

**Comment by genbing99** [comment]
> > Hi, can you upload the code for the network training and testing? Thanks.
> 
> Hello,have you get or implement the code for this network's training and testing?

No, I am still waiting for the reply from authors, seems like the generator loss is not provided.

**Comment by Hazel1021** [comment]
> 

OKay,Thank you for your reply.

**Comment by Hazel1021** [comment]
> > > Hi, can you upload the code for the network training and testing? Thanks.
> > 
> > 
> > Hello,have you get or implement the code for this network's training and testing?
> 
> No, I am still waiting for the reply from authors, seems like the generator loss is not provided.

Okay,thank you for your reply.
